### PR TITLE
Products filter functionalities

### PR DESCRIPTION
**Pull request by dikshadhote**
Following features have been added :

1.Filter by Price using Slider
2.Filter by Category
3.Filter by Rating
4. Sort By high to low and low to high price
5. Clear all filters
image

Go to products page for preview : https://dreamstore-filter-func.netlify.app/

### CODE REVIEW

File: src/components/products/FilterProduct.js [hunk 1]
```diff
@@ -65,7 +94,10 @@ export default function FilterProduct() {
               className="form-check-input"
               type="radio"
               name="sort"
-              checked
+              checked={stateProduct.sortBy === "low-to-high"}
```
Comment (ShreyashMahajan) [inline src/components/products/FilterProduct.js:97]: sortBy can be destructured from stateProduct

File: src/components/products/Products.js [hunk 2]
```diff
@@ -1,36 +1,30 @@
...
     <div>
       <Navbar />
       <div className="productspage-container">
         <FilterProduct />
         <div className="m-2 ">
-          <h3 className="pb-1 pl-2 ">Products</h3>
+          <h3 className="pb-1 pl-2 ">Products ({filteredProducts.length})</h3>
           <div className="d-flex align-items-stretch main-container flex-wrap">
-            {productsList.map(
+            {}
```
Comment (ShreyashMahajan) [inline src/components/products/Products.js]: this can be removed

**Review by ShreyashMahajan — APPROVED** [review]
The code looks neat and understandable. Just look at the suggestion. Good to merge

**Review by khushijohri2001 — APPROVED** [review]
Clean code

**Pull request merged**